src/main/java/no/uib/inf112/core/player/Player.java

```diff
@@ -16,15 +16,18 @@
     public static final int MAX_LIVES = 3;
     public static final int MAX_HEALTH = 10;
     public static final int MAX_PLAYER_CARDS = 5;
+    public static final int MAX_DRAW_CARDS = 9;
```

Comment:
Do we need a `MAX_DRAW_CARDS `when we have `MAX_HEALTH`? Maybe set it to `MAX_HEALTH - 1` to illustrate why it is 9